static/js/src/dynamic-contact-form.js

```diff
@@ -18,10 +18,11 @@
 
     // recaptcha submitCallback
     CaptchaCallback = function() {
-      let recaptchas = document.querySelectorAll("div[class^=g-recaptcha]");
+     let recaptchas = [].slice.call(document.querySelectorAll("div[class^=g-recaptcha]"));
```

Comment:
bad indentation. 
Also if this stuff goes through webpack and the variable isn't redefined, this should be `const`. If not it should be `var`